### DISCUSSION TITLE
Instagram Basic Display not avilable with App-Type Business

### DISCUSSION THREAD

**Discussion by Lapotor** [chatwoot/chatwoot#6623] [Q&A]
I'm currently setting up my instance to work with Facebook and instagram.

Facebook setup works so far.

Now i wanted to test the instagram part as described in the docs guide:
https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/instagram-channel-setup#testing-the-facebook-channel

But you can't add the `Instagram Basic Display` as Product anymore.
But if you use no type you could add it.

![My Products](https://user-images.githubusercontent.com/17144397/223564384-82f807d1-8b1a-47e1-87d6-ee539a7d9f79.png)
![Avilable Products](https://user-images.githubusercontent.com/17144397/223564386-3c4bb9b7-d4ee-4bf7-87d1-3b13d9176394.png)

Are there any workarounds out?

**Comment by Voldemorten** [comment]
@Lapotor did you work something out?
Edit: Here is the [link](https://developers.facebook.com/docs/development/create-an-app/app-dashboard/app-types/) to where it explicitly points out that Instagram Basic Display API is excluded from Business

**Reply by Lapotor** [reply]
No, unfortunately I haven't found anything else for it yet.
Do you have any hints maybe?

**Reply by Voldemorten** [reply]
Nope. I'm trying to work it out myself.

**Reply by Lapotor** [reply]
I haven't tried the `None` type yet maybe this way it will work 🤔

**Reply by Voldemorten** [reply]
>None type apps cannot be created as they are a legacy type.

I don't think that's an option

**Reply by Lapotor** [reply]
Ok I raised an Issue in docs repo:
https://github.com/chatwoot/docs/issues/489

**Comment by sojan-official** [comment]
We have released a dedicated instagram channel using instagram business login in v4.1.0 
ref: https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/instagram-via-instagram-business-login
